frontend/packages/ceph-storage-plugin/src/components/dashboard-page/storage-dashboard/breakdown-card/breakdown-chart.tsx

```diff
@@ -0,0 +1,96 @@
+import * as React from 'react';
+import { Link } from 'react-router-dom';
+import {
+  Chart,
+  ChartAxis,
+  ChartBar,
+  ChartLabel,
+  ChartLegend,
+  ChartStack,
+  ChartThemeColor,
+  ChartTooltip,
+} from '@patternfly/react-charts';
+import { DataPoint } from '@console/internal/components/graphs';
+import { K8sKind } from '@console/internal/module/k8s';
+import { resourcePathFromModel } from '@console/internal/components/utils';
+import { isAvailableBar, getBarRadius } from './utils';
+import './breakdown-card.scss';
+
+const LinkableLegend = (props) => {
```

Comment:
```suggestion
const LinkableLegend = ({ metricModel, datum }) => {
```

and then the return can be inlined